blaze-client/src/main/scala/org/http4s/client/blaze/package.scala

```diff
@@ -2,12 +2,14 @@ package org.http4s
 package client
 
 import cats.effect._
+import fs2.Stream
 
 package object blaze {
 
-  /** Default blaze client
-    *
-    * This client will create a new connection for every request.
-    */
-  def defaultClient[F[_]: Effect]: Client[F] = SimpleHttp1Client(BlazeClientConfig.defaultConfig)
+  def defaultClient[F[_]: Effect]: F[Client[F]] = Http1Client()
+
+  def defaultClientStream[F[_]: Effect]: Stream[F, Client[F]] = Http1Client.stream()
+
+  @deprecated("Use Http1Client instead", "0.18.0-M6")
+  type PooledHttp1Client = Http1Client.type
```

Comment:
Since the signature of `apply` is changing, does it make more sense to leave a deprecated object instead of case class?

Reply:
Yes.

Reply:
The parameters to `PooledHttp1Client.apply` are different than the parameters to `Http1Client`.  Anyone who calls `PooledHttp1Client` is not going to compile.  If we deprecated the old one, we could point them to the new one but let them upgrade at their own pace.